### PR TITLE
Add Giraffe

### PR DESCRIPTION
**Pull request by Palleas**

### CODE REVIEW

File: _data/projects.yml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+- name: Giraffe
+  description: "iOS client for Giphy written in Swift with ReactiveCocoa."
+  highlights: [MVVM, Networking, Protocol Extensions, Unit Testing]
+  type: ios
+  rac_version: 4
+  repo_url: https://github.com/richeterre/SwiftGoal
```
Comment (ikesyo) [inline _data/projects.yml:6]: This seems not to be correct 😅

File: _data/projects.yml [hunk 2]
```diff
@@ -1,3 +1,9 @@
+- name: Giraffe
+  description: "iOS client for Giphy written in Swift with ReactiveCocoa."
+  highlights: [MVVM, Networking, Protocol Extensions, Unit Testing]
```
Comment (ikesyo) [inline _data/projects.yml:3]: I'm not sure about the library, but is that really related to Unit Testing?

File: _data/projects.yml (hunk 1 above)
Comment (Palleas) [inline _data/projects.yml:6]: Woops!
